File: solutions/lua/raindrops/1/raindrops.lua
return function(n)
	local result = ""
	if n % 3 == 0 then
		result = result .. "Pling"
	end

	if n % 5 == 0 then
		result = result .. "Plang"
	end

	if n % 7 == 0 then
		result = result .. "Plong"
	end

	if #result == 0 then
		return n .. ""
	end

	return result
end

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "strconv"

func Convert(number int) string {

	inText := ""
	if number%3 == 0 {
		inText += "Pling"
	}

	if number%5 == 0 {
		inText += "Plang"
	}

	if number%7 == 0 {
		inText += "Plong"
	}

	if inText != "" {
		return inText
	}
	return strconv.Itoa(number)
}
